.prettierrc

```diff
@@ -0,0 +1,6 @@
+tabWidth: 4
+printWidth: 120
```

Comment:
It's just personnal preferences and habits.
80 is the default (mostly because it's the old terminals width)
and using more than that is even NOT recommended AT ALL 
https://prettier.io/docs/en/options.html#print-width
I just feel like I hit it very soon. And with an indentation of four spaces, it does not help :grin: 
I don't split my screen and I like very few lines. Some people like few columns.

ANYWAY I'll re-run it with 80 and we'll see :smiley:

Reply:
> ANYWAY I'll re-run it with 80 and we'll see smiley

Oh you already did! Thank you!

Reply:
Yes, you run into 80 pretty quickly, that's for sure. Honestly, though, there aren't too many things changed in that commit. Just a couple of outliers. Some of them looked odd anyway. 🤷